### PR TITLE
Converted README files from RST to Markdown

### PR DESCRIPTION
**Pull request by andy-maier**

### CODE REVIEW

File: README_PYPI.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+$ pywbemcli connection save --help
+. . .
+```
+
+The following examples build on each other and show a typical sequence
+of exploration of a WBEM server. For simplicity, they all operate
+against the default namespace of the server, and use a persistent
+connection definition for the server:
+
+-   Add a persistent connection definition named `conn1` for the WBEM
```
Comment (KSchopmeyer) [inline README_PYPI.md:67]: change to say persistent WBEM server connection

File: README_PYPI.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    ------  -----------------  -----------  ------  ---------  ----------  --------  ----------  ---------  --------------------------------------
+    assoc1                     root/cimv2                  30              True                             tests/unit/simple_assoc_mock_model.mof
+    conn1   https://localhost  root/cimv2   user           30  True        False
+    ```
+
+-   Show the class tree, using the previously added connection
+    definition `assoc1`:
+
+    ``` text
+    $ pywbemcli -n assoc1 class tree
```
Comment (KSchopmeyer) [inline README_PYPI.md:109]: add "where -n is a pywbemcli persistent connection name (see pywbemcli connection)

**Review by KSchopmeyer — COMMENTED** [review]
Made a couple of comments to clarify documentation.  Other than that, approved.

File: README_PYPI.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+-   pywbemcli also supports mocked WBEM servers in memory, that are
+    preloaded with CIM objects defined in MOF files. Add a persistent
+    connection definition named `assoc1` to a mock server using one of
+    the MOF files provided in the repo:
+
+    ``` text
+    $ pywbemcli -m tests/unit/simple_assoc_mock_model.mof connection save assoc1
+    ```
+
+-   List the persistent connection definitions:
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply README_PYPI.md:83]: DONE

File: README_PYPI.md (hunk 2 above)
Comment (andy-maier) [reply README_PYPI.md:109]: We already clarify that partially in the sentence before the example. Added the `-n` option there:
```
Show the class tree, using the previously added WBEM server connection
definition `assoc1` (specified with the `-n` option):
```

File: README_PYPI.md (hunk 1 above)
Comment (andy-maier) [reply README_PYPI.md:67]: It already mentions "WBEM server" in the second half of the sentence:
```
Add a persistent connection definition named `conn1` for the WBEM
server to be used,
```

**Pull request merged**